sympy/tensor/tensor.py

```diff
@@ -28,7 +28,7 @@
 If there is a (anti)symmetric metric, the indices can be raised and
 lowered when the tensor is put in canonical form.
 """
-
+from future import __annotations__
```

Comment:
You got this backwards. Should be `from __future__ import annotations`.

Reply:
Gah! Thanks!